src/ripple/basics/StringUtilities.h

```diff
@@ -48,6 +48,24 @@ template <class Iterator>
 std::optional<Blob>
 strUnHex(std::size_t strSize, Iterator begin, Iterator end)
 {
+    static constexpr std::array<int, 256> const unxtab = []() {
```

Comment:
This file now needs a `#include <array>`.  Please add that near the top of the file like this...
```
...
#include <boost/utility/string_view.hpp>
#include <array>
#include <optional>
...
```